Clients/src/presentation/components/Modals/ComplianceFeedback/ComplianceFeedback.tsx

```
@@ -0,0 +1,229 @@
+            setBulleted(false); // Disable bullet list if active
+          }
+          break;
+        default:
+          break;
+      }
+      setText(updatedText);
+    };
+
+    const handleFileUpload = (e: React.ChangeEvent<HTMLInputElement>) => {
+      if (e.target.files && e.target.files[0]) {
+        setFile(e.target.files[0]);
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)